bundles/org.eclipse.ui.navigator.resources/src/org/eclipse/ui/internal/navigator/resources/actions/ResourceMgmtActionProvider.java

```diff
@@ -139,19 +139,23 @@ public void fillContextMenu(IMenuManager menu) {
 				hasBuilder = false;
 			}
 		}
+
```

Comment:
the change would look better if it would avoid unnecessary new lines added.

Reply:
Fixed in the commits added today